protocol-specs/eth2_spec_tests/tests/general/phase0/bls/aggregate/small/aggregate_na_signatures/data.yaml

```diff
@@ -0,0 +1,2 @@
+input: []
```

Comment:
this case should be skipped.

Reply:
We don't have any custom implementation for it. Error been thrown from the library as `EMPTY_AGGREGATE_ARRAY` and we return `false` for it. I think its good to have this case.

Reply:
ok, I missed that your test is interpreting empty expected output as INVALID.